README.md

```diff
@@ -83,7 +83,26 @@ var menu = document.querySelector('.menu');
 menu.label = 'Sortieren';
 ```
 
-*NOTE: Undocumented attributes are assumed to be HTML and will be applied to the root element of the custom tag.*
+#### Pass-Through Attributes
+
+HTML attributes can be used on any component, and they will be passed through to the most prominent tag of the component. The most prominent tag is usually the root, but individual components will note if it varies for specific cases.
+
+Example of static usage:
+```marko
+<ebay-button id="my-button"/>
+```
+
+For using pass-through attributes dynamically, they should be sent through the `htmlAttributes` attribute:
+```marko
+<!-- data.htmlAttributes = {id: 'my-button'} -->
+<ebay-button htmlAttributes=data.htmlAttributes/>
```

Comment:
@yomed We should have the attribute name as all lowercase right (`htmlattributes`)? Our original coding guideline is to use all HTML attributes as lowercase. You can either have it `htmlattributes` or `html-attributes`